code/game/gamemodes/meteor/meteor.dm

```diff
@@ -69,6 +69,7 @@
 		universe.OnExit()
 
 	var/datum/universal_state/meteor_storm/meteor_storm = new /datum/universal_state/meteor_storm
+	universe = meteor_storm
```

Comment:
Note that that's the proper approach but if you'd have even tried you could've atleast just passed `FALSE` as third argument to SetUniversalState to tell it NOT to call on enter directly.